GCEWindowsAgent/diagnostics.go

```diff
@@ -27,8 +26,7 @@ import (
 const diagnosticsCmd = `C:\Program Files\Google\Compute Engine\diagnostics\diagnostics.exe`
 
 var (
-	diagnosticsRegKey   = "Diagnostics"
```

Comment:
Since your here anyway can you just drop this manager completly, if you want to move this to another PR thats OK to